lib/chef_zero/endpoints/organization_policies_endpoint.rb

```diff
@@ -0,0 +1,99 @@
+
+module ChefZero
+  module Endpoints
+    # /organizations/NAME/policies
+    class OrganizationPoliciesEndpoint < RestBase
+      def set_opts
+        [ :create_dir, :data_store_exceptions ]
```

Comment:
What do these opts do?  I'm guessing `:create_dir` will create the `policies` directory on a post if it doesn't exist yet